service-base/src/test/java/org/eclipse/hono/service/plan/PrometheusBasedResourceLimitChecksTest.java

```diff
@@ -302,9 +318,12 @@ public void testMessageLimitUsesValueFromCache(final TestContext ctx) {
         final TenantObject tenant = TenantObject.from("tenant", true).setResourceLimits(limitsConfig);
 
         limitChecksImpl.isMessageLimitReached(tenant, incomingMessageSize)
-                .setHandler(ctx.asyncAssertSuccess(b -> {
-                    ctx.assertEquals(Boolean.TRUE, b);
-                    verify(webClient, never()).get(eq(DEFAULT_PORT), eq(DEFAULT_HOST), anyString());
+                .setHandler(ctx.succeeding(response -> {
+                    ctx.verify(() -> {
+                        assertEquals(Boolean.TRUE, response);
```

Comment:
assertTrue?

Reply:
My bad I assumed that it was only 2 places as mentioned :-(
I have made changes to all the places with _assertTrue_ or _assertFalse_.